OpenRA.Game/Activities/Activity.cs

```diff
@@ -39,6 +39,7 @@ protected Activity ChildActivity
 		}
 
 		public Activity NextActivity { get; protected set; }
+		public Activity CurrentChildActivity { get { return ChildActivity; } }
```

Comment:
IMO it would be better to create a `DebugLabel` property that returns the label for the current activity + any children instead of exposing this.

`RenderDebugState` would then call this on `CurrentActivity` instead of doing it itself.